### ISSUE TITLE
Use shell script for cell-type-wilms-tumor-06 workflow

### ISSUE THREAD

**Issue by sjspielman** [AlexsLemonade/OpenScPCA-analysis#816]
This issue is based on this discussion: https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/814#issuecomment-2415199088

Briefly, we have discovered that calling `system()` from within an Rscript is actually not a robust strategy for running the module workflow, since individual steps that fail don't cause the entire workflow to error. Therefore, it would be best to instead use a shell script to run the workflow, which for consistency we'll call `00_run_workflow.sh`.

I'm also opening this issue so @maud-p is aware - for now, you can feel free to please continue working with your existing `00_run_workflow.R` script just as you have been! While I'm working in parallel, I will not modify `00_run_workflow.R`. Once all code related to #810 is merged into `main`, you can file a PR to formally remove `00_run_workflow.R` (I can also help to remove this file, but I think it's best if you do it so that you have no confusion when it's time to stop using that script!). I will make sure to let you know when code has been merged and we're ready for that next step, and I can assist as needed!

Noting we should not close this issue out until the workflow is formally using the new shell script from `main`.

**Comment by sjspielman** [comment]
With #883, this new script was merged into main, and the old R script was removed.
